app/views/idv/shared/_verify.html.erb

```diff
@@ -42,54 +69,62 @@ locals:
+                <dd class="display-inline margin-left-0"> <%= pii[:city] %> </dd>
+            </div>
+            <div>
+                <dt class="display-inline"> <%= t('idv.form.state') %>: </dt>
+                <dd class="display-inline margin-left-0"> <%= pii[:state] %> </dd>
+            </div>
+            <div>
+                <dt class="display-inline"> <%= t('idv.form.zipcode') %>: </dt>
+                <dd class="display-inline margin-left-0"> <%= pii[:zipcode] %> </dd>
+            </div>
+          </div>
+        </dl>
+        <div class='grid-auto'>
+          <%= button_to(
+                step_url.call(step: :redo_address),
+                method: :put,
+                class: 'usa-button usa-button--unstyled',
+                'aria-label': t('idv.buttons.change_address_label'),
+              ) { t('idv.buttons.change_label') } %>
         </div>
-      <% end %>
-      <div>
-        <dt class="display-inline"> <%= t('idv.form.city') %>: </dt>
-        <dd class="display-inline margin-left-0"> <%= pii[:city] %> </dd>
-      </div>
-      <div>
-        <dt class="display-inline"> <%= t('idv.form.state') %>: </dt>
-        <dd class="display-inline margin-left-0"> <%= pii[:state] %> </dd>
-      </div>
-      <div>
-        <dt class="display-inline"> <%= t('idv.form.zipcode') %>: </dt>
-        <dd class="display-inline margin-left-0"> <%= pii[:zipcode] %> </dd>
-      </div>
-    </dl>
-    <div class='grid-auto'>
-      <%= button_to(
-            step_url.call(step: :redo_address),
-            method: :put,
-            class: 'usa-button usa-button--unstyled',
-            'aria-label': t('idv.buttons.change_address_label'),
-          ) { t('idv.buttons.change_label') } %>
     </div>
-  </div>
   <div class="grid-row grid-gap grid-gap-2 padding-top-1">
     <div class='grid-col-fill'>
-      <%= t('idv.form.ssn') %>:
-      <%= render(
-            'shared/masked_text',
-            text: SsnFormatter.format(pii[:ssn]),
-            masked_text: SsnFormatter.format_masked(pii[:ssn]),
-            accessible_masked_text: t(
-              'idv.accessible_labels.masked_ssn',
-              first_number: pii[:ssn][0],
-              last_number: pii[:ssn][-1],
-            ),
-            toggle_label: t('forms.ssn.show'),
-          ) %>
+    <% if capture_secondary_id_enabled %>
+      <strong><%= t('headings.ssn') %></strong>
+    <% end %>
+      <div>
+        <%= t('idv.form.ssn') %>:
+        <%= render(
+              'shared/masked_text',
+              text: SsnFormatter.format(pii[:ssn]),
+              masked_text: SsnFormatter.format_masked(pii[:ssn]),
+              accessible_masked_text: t(
+                'idv.accessible_labels.masked_ssn',
+                first_number: pii[:ssn][0],
+                last_number: pii[:ssn][-1],
+              ),
+              toggle_label: t('forms.ssn.show'),
+            ) %>
+      </div>
```

Comment:
Same note:

```suggestion
      <%= t('idv.form.ssn') %>:
      <%= render(
            'shared/masked_text',
            text: SsnFormatter.format(pii[:ssn]),
            masked_text: SsnFormatter.format_masked(pii[:ssn]),
            accessible_masked_text: t(
              'idv.accessible_labels.masked_ssn',
              first_number: pii[:ssn][0],
              last_number: pii[:ssn][-1],
            ),
            toggle_label: t('forms.ssn.show'),
          ) %>
```

Reply:
this one we need to keep the div otherwise the next line moves onto the same line as the section title